packages/ui-test-utils/src/utils/generateComponentExamples.ts

```diff
@@ -58,7 +96,7 @@ export type ExamplesPage<Props> = {
 export type Example<Props> = {
   Component: ComponentType
   componentProps: Partial<Props>
-  exampleProps: Partial<Props>
+  exampleProps: Record<string, any> // actually Partial<ViewProps>
```

Comment:
Shouldn't we change the type to `Partial<ViewProps>`then? Or it throws errors then?

Reply:
Nope, it creates a circular dependency